doc/s9s-cluster.1

```diff
@@ -171,6 +171,51 @@ mys9s cluster \\
     --wait
 .fi
 
+.TP
+.B \-\-upgrade-to-version
+The new newer major version to upgrade too. Without this,
+only minor upgrade will be done.
+
+.B EXAMPLE
+.nf
+mys9s cluster \\
+    --upgrade-cluster \\
+    --provider-version=12 \\
+    --upgrade-to-version=12 \\
```

Comment:
the command to upgrade from `11` to `12` major version, right?
Should `--provider-version` and `--upgrade-to-version` versions be the same major version to upgrade to?

Reply:
One of these will be dropped. These are two suggestions. --provider-version is already there, but the other name is more descriptive. So it might worth to introduce it.

Reply:
Agree about `--upgrade-to-version` name.
`--upgrade-cluster` option with `--upgrade-to-version` option looks better for me.

Reply:
So we are replacing `--provider-version` with `--upgrade-to-version`, right?